src/components/v2/Layout/Sidebar.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import Toolbar from '@mui/material/Toolbar';
+import { NavLink } from 'react-router-dom';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import Drawer from '@mui/material/Drawer';
+import { uid } from 'react-uid';
+import Typography from '@mui/material/Typography';
+import { useTheme } from '@mui/material';
+import { ReactComponent as XVSIcon } from '../../../assets/img/xvs.svg';
+import { ReactComponent as Logo } from './assets/logo.svg';
+
+const menuItems = [
+  {
+    href: '/dashboard',
+    text: 'Dashboard',
+    icon: <XVSIcon width={20} height={20} />,
+  },
+  {
+    href: '/vote',
+    text: 'Vote',
+    icon: <XVSIcon width={20} height={20} />,
+  },
+  {
+    href: '/xvs',
+    text: 'XVS',
+    icon: <XVSIcon width={20} height={20} />,
+  },
+  {
+    href: '/market',
+    text: 'Market',
+    icon: <XVSIcon width={20} height={20} />,
+  },
+  {
+    href: '/vault',
+    text: 'Vault',
+    icon: <XVSIcon width={20} height={20} />,
```

Comment:
We probably want to have these values be a constant in the theme so we have consistent icons across the app

Reply:
seems like this values may be constants just in scope of sidebar.
in other parts of app may be other sizes for icons